lib/BackgroundJob/Tasks/ImageProcessingTask.php

```diff
@@ -388,7 +388,7 @@ private function calculateMaxImageArea(): int {
 		// Based on amount on memory PHP have, we will determine maximum amount of image size that we need to scale to.
 		// This reasoning and calculations are all based on analysis given here:
 		// https://github.com/matiasdelellis/facerecognition/wiki/Performance-analysis-of-DLib%E2%80%99s-CNN-face-detection
-		$maxImageArea = intval((0.75 * $allowedMemory) / 1024); // in pixels^2
```

Comment:
I have pending return this somewhere more coherent.

According to your study, the photo analysis, linearly requires 1GB of ram for each MB de area in px. Which is very easy, but when cut it here is difficult to evaluate.. 

I think that cutting this value is important only when we use the values calculated from the php.ini But in configured values we must trust them.